src/main/java/team/creative/creativecore/common/util/type/list/Pair.java

```diff
@@ -36,7 +36,7 @@ public int hashCode() {
     public boolean is(K key) {
         if (this.key != null)
             return this.key.equals(key);
-        return this.key == key;
```

Comment:
This basically checks if the other key is null. Don't see a need to change it, of course you can go with == null as well.

Reply:
That's for code readbility